asb-native/src/main/java/org/ballerinax/asb/util/ASBUtils.java

```diff
@@ -174,6 +174,11 @@ public static Map<String, Object> toObjectMap(BMap<BString, Object> map) {
         return returnMap;
     }
 
+    @SuppressWarnings("unchecked")
```

Comment:
```suggestion
    @SuppressWarnings("unchecked")
```
Is this required?

Reply:
Added it cos it suppresses the unnecessary compiler warnings.

Reply:
Fixed with https://github.com/ballerina-platform/module-ballerinax-azure-service-bus/pull/137/commits/4f73fff235e0a1f53cc33cd1344116af0d702c7b